tools/vendoring/patches/cpuinfo.patch

```diff
@@ -1,44 +0,0 @@
-diff --git b/conda/_vendor/cpuinfo/cpuinfo.py a/conda/_vendor/cpuinfo/cpuinfo.py
-index 10be23113..39abe9778 100644
---- b/conda/_vendor/cpuinfo/cpuinfo.py
-+++ a/conda/_vendor/cpuinfo/cpuinfo.py
-@@ -550,13 +550,16 @@ def parse_arch(raw_arch_string):
- 	raw_arch_string = raw_arch_string.lower()
- 
- 	# X86
--	if re.match('^i\d86$|^x86$|^x86_32$|^i86pc$|^ia32$|^ia-32$|^bepc$', raw_arch_string):
-+	if re.match(r'^i\d86$|^x86$|^x86_32$|^i86pc$|^ia32$|^ia-32$|^bepc$', raw_arch_string):
- 		arch = 'X86_32'
- 		bits = 32
- 	elif re.match('^x64$|^x86_64$|^x86_64t$|^i686-64$|^amd64$|^ia64$|^ia-64$', raw_arch_string):
- 		arch = 'X86_64'
- 		bits = 64
- 	# ARM
-+	elif re.match('^arm64$|^arm64[a-z]$|^arm64-[a-z]$', raw_arch_string):
-+		arch = 'ARM_8'
-+		bits = 64
```

Comment:
I think this section is currently not covered by cpuinfo when looking at the regex patterns in https://github.com/workhorsy/py-cpuinfo/blob/220d5b05e26160474a384c8803e59488d4292228/cpuinfo/cpuinfo.py#L804-L806.

@chenghlee do you happen to know if we actually need `^arm64$|^arm64[a-z]$|^arm64-[a-z]$` or could we rely on `^armv8-a|aarch64|arm64$`?

Reply:
I don't think `arm64-?[a-z]` are a thing (looking at [this list from Wikipedia](https://en.wikipedia.org/wiki/Uname))?

Reply:
The regex pattern used in our patch looks like a copy/paste of the `armv7`/`armv8` patterns.

Reply:
Yeah, @katietz confirmed your suspicion that this wasn't correct in the old patch and cpuinfo is up-to-date